src/app.js

```diff
@@ -1,17 +1,20 @@
 const express = require('express');
-
+const swaggerUi = require('swagger-ui-express');
 const routes = require('./routes');
 
 // Database connection
 require('./helpers/databaseConnection');
 
 const { errorHandler, notFound } = require('./middlewares');
 
+const openApiDocumentation = require('../APIs.json');
+
 // Initialize server
 const app = express();
 
 // App middleware
 app.use(express.json());
+app.use('/apiDocs', swaggerUi.serve, swaggerUi.setup(openApiDocumentation));
```

Comment:
Move all code to separate router file and include in routes.js

Reply:
Moved Swagger code to routes.js.

Reply:
Alright, that should do it. Please review again.
More modular code, added a structure to maintain API versions as well.